source/blog/2017-07-21-rpm-ostree-v2017.7-released.html.md

```diff
@@ -63,9 +63,9 @@ layered:
 # rpm-ostree status
 State: idle
 Deployments:
-* vmcheck
-                 Timestamp: 2017-07-24 16:07:25
-                    Commit: 20431e5b436c495e3b974bfaf231dac546ee76d3596dc97d6fca7282bdb40506
+* fedora-atomic:fedora-atomic/25/x86_64/docker-host
```

Comment:
Thinking more on this, it doesn't make much sense either to use the current prod, since the rpm-ostree version blogged about *isn't* in that commit. E.g. the output above isn't what users would see since 26.85 still has v2017.6 in it, which prints `GPGSignature` over 3 lines. I guess the correct thing is to rebase to the testing ref, which is what I advertise at the top of the blog post.